common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -28,8 +28,9 @@ run() {
 killall_program() {
     PROGRAM_NAME=$1
     KILL_SIG=${2:-TERM}
+    TIMEOUT=10
     echo "killing $PROGRAM_NAME ($KILL_SIG)"
-    start-stop-daemon -K -s "$KILL_SIG" -x "$PRPLMESH_BIN_DIR"/"$PROGRAM_NAME" > /dev/null 2>&1
+    start-stop-daemon -K -s "$KILL_SIG" -R "$TIMEOUT" -x "$PRPLMESH_BIN_DIR"/"$PROGRAM_NAME" > /dev/null 2>&1
```

Comment:
Unfortunately this option is not available on all platforms (the current prplwrt image doesn't have it either).

That's why we need to do the same ourselves.